packages/terra-abstract-modal/tests/wdio/abstract-modal-spec.js

```diff
@@ -0,0 +1,115 @@
+/* global before, browser, Terra */
+
+describe('Abstract Modal', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+
+  describe('Append Class', () => {
+    before(() => browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/abstract-modal-append-class'));
+
+    Terra.should.matchScreenshot({ selector: 'div[role=document]' });
+    Terra.should.beAccessible();
+  });
+
+  describe('Content Overflow', () => {
+    before(() => browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/abstract-modal-content-overflow'));
+    const overflowModal = { height: 1684, width: 1870 };
+
+    Terra.should.matchScreenshot({ viewports: [overflowModal], selector: 'div[role=document]' });
+    Terra.should.beAccessible({ viewports: [overflowModal] });
+    Terra.should.themeCombinationOfCustomProperties({
+      testName: 'themed',
+      properties: {
+        '--terra-abstract-modal-vertical-inset': '50px',
+        '--terra-abstract-modal-horizontal-inset': '50px',
+      },
+      selector: 'div[role=document]',
+      viewports: [overflowModal],
+    });
+  });
+
+  describe('Custom Props', () => {
+    before(() => browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/abstract-modal-custom-props'));
+    const viewport = Terra.viewports('medium')[0];
+
+    Terra.should.matchScreenshot({ viewports: [viewport], selector: 'div[role=document]' });
```

Comment:
Rather than taking a screenshot, this test should check for the custom props, looks like the ID attribute is the only custom prop set on this. This test could also be replaced with a jest test.